### DISCUSSION TITLE
Ficheiro .xsd

### DISCUSSION THREAD

**Discussion by JudsonArtur** [assoft-portugal/SAF-T-AO#156] [General]
O ficheiro .xsd nós é que criamos com orientação do estado, ou só seguimos o que foi criado.

**Comment by cryptolopes** [comment]
Estamos a usar este.

**Reply by JudsonArtur** [reply]
Mas o documento .xsd serve apenas para testar  o SAFT localmente?

**Reply by cryptolopes** [reply]
O schema contém as regras de validação de estrutura do SAF-T.

**Reply by JudsonArtur** [reply]
Obrigado.
